### ISSUE TITLE
Uncaught ReferenceError: data is not defined on @version 3.0.2

### ISSUE THREAD

**Issue by delanthear** [jsanahuja/jquery.instagramFeed#122]
Apologies if this is a really numpty issue.  I've inherited a site running a very old version of this script (1.2.1) and I'm trying to get it running again. The last time I had really anything to do with Javascript was in the early 2000s! :)

I've swapped out the old code for 3.0.2 and replace the host variable with the proxy version suggested here: https://github.com/jsanahuja/jquery.instagramFeed/issues/103#issuecomment-788260968

but I'm getting:
```
vendor.js?ver=5.6.1:1706 Uncaught ReferenceError: data is not defined
    at Object.success (vendor.js?ver=5.6.1:1706)
    at c (jquery.min.js?ver=3.5.1:2)
    at Object.fireWith [as resolveWith] (jquery.min.js?ver=3.5.1:2)
    at l (jquery.min.js?ver=3.5.1:2)
    at XMLHttpRequest.<anonymous> (jquery.min.js?ver=3.5.1:2)
```

1706 for me is: 
```
    function request_data(url, type, tries, callback, autoFallback, googlePrefix){
        var prefixedUrl;
        if(autoFallback && googlePrefix){
            prefixedUrl = 'https://images' + ~~(Math.random() * 3333) + '-focus-opensocial.googleusercontent.com/gadgets/proxy?container=none&url=' + url;
        }
        $.get(prefixedUrl || url, function(response){
            data = parse_response(type, response);   <---
```

I guess this might not be related to this script and is something else loaded on the site, but some clues would be helpful if you have any!  

Thanks!

**Comment by delanthear** [comment]
er.. I just put a "var" before data on that line and it's now returning stuff.  😕

**Comment by mahotilo** [comment]
@jsanahuja,
I think we need this `var` in https://github.com/jsanahuja/jquery.instagramFeed/blob/559fc6c8ecb63b9c57974a4aa5f2ec6dbfcde99f/jquery.instagramFeed.js#L150
don't we?

**Comment by jsanahuja** [comment]
> @jsanahuja,
> I think we need this `var` in https://github.com/jsanahuja/jquery.instagramFeed/blob/559fc6c8ecb63b9c57974a4aa5f2ec6dbfcde99f/jquery.instagramFeed.js#L150
> don't we?

Definitely, feel free to do it if you want :)

**Comment by jsanahuja** [comment]
Changes published in [3.0.3 release](https://github.com/jsanahuja/jquery.instagramFeed/releases/tag/3.0.3)
